game/game.go

```diff
@@ -442,29 +432,32 @@ type Game struct {
 	Closed   bool // Game is no longer joinable.
 	Finished bool // Game has reached its end.
 
-	Desc                          string           `methods:"POST,PUT" datastore:",noindex"`
-	Variant                       string           `methods:"POST"`
-	PhaseLengthMinutes            time.Duration    `methods:"POST,PUT"`
-	NonMovementPhaseLengthMinutes time.Duration    `methods:"POST,PUT"`
-	MaxHated                      float64          `methods:"POST"`
-	MaxHater                      float64          `methods:"POST"`
-	MinRating                     float64          `methods:"POST"`
-	MaxRating                     float64          `methods:"POST"`
-	MinReliability                float64          `methods:"POST"`
-	MinQuickness                  float64          `methods:"POST"`
-	Private                       bool             `methods:"POST"`
-	NoMerge                       bool             `methods:"POST"`
-	DisableConferenceChat         bool             `methods:"POST,PUT"`
-	DisableGroupChat              bool             `methods:"POST,PUT"`
-	DisablePrivateChat            bool             `methods:"POST,PUT"`
-	NationAllocation              AllocationMethod `methods:"POST"`
-	Anonymous                     bool             `methods:"POST"`
-	LastYear                      int              `methods:"POST,PUT"`
-	SkipMuster                    bool             `methods:"POST,PUT"`
-	ChatLanguageISO639_1          string           `methods:"POST,PUT"`
-	GameMasterEnabled             bool             `methods:"POST"`
-	RequireGameMasterInvitation   bool             `methods:"POST,PUT"`
-	DiscordWebhooks               *DiscordWebhooks `methods:"POST" datastore:",noindex"`
```

Comment:
Just removing the `*` should make it work I think.

Reply:
Done. Tested locally and it worked. thanks!

Reply:
But wait, doesn't that mean that it's a required field? I don't want to break existing create game ux

Reply:
Go will just default it to the empty value, which in this case will be two empty strings. And datastore works the same way, so nothing will break :D

All old games will just have empty values in those fields.